### PR TITLE
Clarify cellular connection configuration via dbus

### LINKED ISSUE #555: Clarify that cellular connections can also be added via the dbus interface
Front logo [Front conversations](https://app.frontapp.com/open/top_25mt)

### PR DESCRIPTION
**Pull request by notnamed**
Fixes #555 by clarifying and linking to DBUS section for configuring cellular connections at runtime

Change-type: minor
Signed-off-by: Jordan Cooks <jordan@balena.io>

### CODE REVIEW

File: shared/network/2.x.md [hunk 1]
```diff
@@ -264,7 +264,7 @@ This will create a WPA2/rsn hotspot, in case you need WPA protocol replace `prot
 
 ## Cellular Modem Setup
 
-For cellular or GSM based connections, {{ $names.os.lower }} makes use of **[ModemManager][modemmanager-link]** and is configured in much the same way as WiFi connections are configured. Currently, it is only possible to add a cellular configuration by adding a connection file to `/resin-boot/system-connections` in the downloaded OS `.img` file, but in the future cellular connections will be configurable from the dashboard at download time.
+For cellular or GSM based connections, {{ $names.os.lower }} makes use of **[ModemManager][modemmanager-link]** and is configured in much the same way as WiFi connections are configured. The connection profile can either be specified in your application code (see [Changing the Network at Runtime](#changing-the-network-at-runtime) below), or you can add a cellular configuration by adding a connection file to `/resin-boot/system-connections` in the downloaded OS `.img` file. In the future, cellular connections will be configurable from the dashboard at download time.
```
Comment (vipulgupta2048) [inline shared/network/2.x.md:267]:
```suggestion
For cellular or GSM based connections, {{ $names.os.lower }} makes use of **[ModemManager][modemmanager-link]** and is configured in much the same way as WiFi connections are configured. The connection profile can either be specified in your application code (see [Changing the Network at Runtime](#changing-the-network-at-runtime) below), or you can add a cellular configuration by adding a connection file to `/resin-boot/system-connections` in the downloaded {{ $names.os.lower }} `.img` file. In the future, cellular connections will be configurable from the dashboard at time of adding a device and downloading a {{ $names.os.lower }} image.
```

**Comment by vipulgupta2048** [comment]
Hey @notnamed did you forget to push the commit after making the suggested changes?

**Comment by notnamed** [comment]
@vipulgupta2048 using the GH web interface here, I accepted your changes - I don't have any commits to push. Not sure what I'm meant to do here?

File: shared/network/2.x.md (hunk 1 above)
Comment (vipulgupta2048) [inline shared/network/2.x.md:267]:
```suggestion
For cellular or GSM based connections, {{ $names.os.lower }} makes use of **[ModemManager][modemmanager-link]** and is configured in much the same way as WiFi connections are configured. The connection profile can either be specified in your application code (see [Changing the Network at Runtime](#changing-the-network-at-runtime) below), or you can add a cellular configuration by adding a connection file to `/resin-boot/system-connections` in the downloaded {{ $names.os.lower }} `.img` file. In the future, cellular connections will be configurable from the dashboard at time of adding a device and downloading a {{ $names.os.lower }} image.
```

**Comment by vipulgupta2048** [comment]
@notnamed Now, it's been added. I have made another commit and will be squashing these up.

**Review by vipulgupta2048 — APPROVED** [review]

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Pull request merged**